ddtrace/debugging/__init__.py

```diff
@@ -0,0 +1,25 @@
+"""
+DD Debugger
```

Comment:
Should we drop the "DD"? 
```suggestion
Debugger
```